source/sessions/driver-sessions.rst

```diff
@@ -1143,17 +1142,21 @@ Test Plan
     * Attach a command started listener that collects each command's lsid
     * Initiate the following concurrent operations
 
-      * insertOne
-      * deleteOne
-      * updateOne
-      * bulkWrite ``[ { updateOne } ]``
-      * findOneAndDelete
-      * findOneAndUpdate
-      * findOneAndReplace
-      * find
+      * ``insertOne({ }),``
+      * ``deleteOne({ }),``
+      * ``updateOne({ }, { $set: { a: 1 } }),``
+      * ``bulkWrite([{ updateOne: { filter: { }, update: { $set: { a: 1 } } } }]),``
+      * ``findOneAndDelete({ }),``
+      * ``findOneAndUpdate({ }, { $set: { a: 1 } }),``
+      * ``findOneAndReplace({ }, { a: 1 }),``
+      * ``find().toArray()``
+
+    * Wait for all operations to complete successfully
+    * Assert the following across at least 5 retries of the above test:
+
+      * Drivers MUST assert that exactly one session is used for all operations at least once across the retries of this test.
```

Comment:
This test should mention why we might need to repeat the test since the issue is subtle:
> Note that it's possible, although rare, for >1 server session to be used because the session is not released until after the connection is checked in.